### DISCUSSION TITLE
BTT OCTOPUS PRO 429 version build failed

### DISCUSSION THREAD

**Discussion by ashokmanker** [grblHAL/STM32F4xx#177] [General]
Please help me compiling grblhal for BTT octopus pro 429 based board. 

`.pio/build/btt_skr_2_tmc5160/src/driver.o: In function `driver_init':
driver.c:(.text.driver_init+0x51c): undefined reference to `board_init'`

getting this error.

**Comment by datdadev** [comment]
I see there is one listed in the issue that using platformio to build this project
